armory/__main__.py

```diff
@@ -170,18 +176,24 @@ def _root(parser):
 # Config
 
 
-def _set_gpus(config, use_gpu, gpus):
+def _set_gpus(config, use_gpu, no_gpu, gpus):
     """
     Set gpu values from parser in config
     """
+    if (use_gpu or gpus) and no_gpu:
+        raise ValueError("no_gpu cannot be set with use_gpu or gpus!")
+
     if gpus:
         if not use_gpu:
             logger.info("--gpus field specified. Setting --use-gpu to True")
             use_gpu = True
         config["sysconfig"]["gpus"] = gpus
+
     if use_gpu or "use_gpu" not in config["sysconfig"]:
         # Override if use_gpu, otherwise if config exists, leave config setting in place
         config["sysconfig"]["use_gpu"] = use_gpu
+    elif no_gpu:
+        config["sysconfig"]["use_gpu"] = not no_gpu
```

Comment:
Let's just set this to `False`. It's a bit confusing as it is. The previous line utilizes `use_gpu` for the case that it's not specified in the sysconfig, which means it can be set to True or False, but here the only valid value is False.